lib/src/screens/send/widgets/extract_address_from_parsed.dart

```diff
@@ -56,6 +56,13 @@ Future<String> extractAddressFromParsed(
       profileImageUrl = parsedAddress.profileImageUrl;
       profileName = parsedAddress.profileName;
       break;
+    case ParseFrom.thorChain:
+      title = S.of(context).address_detected;
+      content = S.of(context).extracted_address_content('${parsedAddress.name} (ThorChain)');
+      address = parsedAddress.addresses.first;
+      profileImageUrl = parsedAddress.profileImageUrl;
+      profileName = parsedAddress.profileName;
```

Comment:
those 2 are not valid for thorchain and will throw an exception most probably if they are not handled correctly in the app

Reply:
This case is handled in the app, but it is indeed unnecessary in the case of ThorChain.